apps/docs/src/content/examples/animation/Complex sprite scene.mdx

```diff
@@ -0,0 +1,132 @@
+`demonSpriteMeta` object, which describes the layout and animation details of the spritesheet.
+
+In this case, the spritesheet image is arranged in a grid of 4 rows and 8 columns, so the `type` is set to `'rowColumn'`,
+`height` to `4` (indicating the number of rows), and `width` to `8` (representing the number of columns).
+The `animations` property is an array, where each element represents a separate animation with a `name` and a `frameRange`.
+
+For detailed information on defining animations and using frame ranges, refer to the [Spritesheet builder section](/docs/reference/extras/instanced-sprite#spritesheetmetadata)
+
+### Adding component to the scene
+
+```svelte title=Flyer.svelte
+{#await flyerSheetbuilder.spritesheet then spritesheet}
+  <InstancedSprite
+    count={20000}
+    {spritesheet}
+  >
+	<!-- User component for updating instances -->
+    <UpdaterFlying /> /
+	<!-- -->
+  </InstancedSprite>
+{/await}
+```
+
+We add `<InstancedSprite>` to the scene with a `count` `spritesheet` - the only required props. Spritesheet is a result of the promise from the previous step.
+
+To add the `<InstancedSprite>` component to the scene, you need to specify at least two essential properties:
+`count` and `spritesheet`. The `spritesheet` property is the object obtained as the result of awaiting the Promise of the `buildSpritesheet` function called earlier.
+
+
+### Updating instances
+
+In our example, the user made `<UpdaterFlying>` component is responsible for updating sprites.
+This component leverages the `useInstancedSprite()` hook, which makes it easy to access and
+adjust sprite properties such as position and animation.
+
+To update sprite instances, we utilize the `useTask` hook. Inside, a loop iterates over the IDs
+of all instances, applying updates to their positions and assigning the fly animation to each.
+This description is simplified for brevity, this is where you'd have your complex movement or game logic.
+A working example, demonstrating basic random movement, is available in the source of the live example for this component
+(UpdaterFlying.svelte, UpdaterWalking.svelte, UpdaterFlyingHook.svelte).
+
+
+```svelte title=UpdaterFlying.svelte
+<script lang="ts">
+	import { useTask } from '@threlte/core'
+	import { useInstancedSprite } from '@threlte/extras'
+
+	const { updatePosition, count, animationMap, sprite } = useInstancedSprite()
+
+	useTask(()=>{
+		for(let i=0; i<count;i++){
+			updatePosition(i, [0,0,0])
+			sprite.animation.setAt(i, 'fly')
+		}
+	})
+</script>
+```
+
+## Various approaches
+
+This section goes over each component used in the scene and provides a short explanation of different approaches used with `<InstancedSprite/>` component.
+Every component is designed differently, aimed to present varied approaches to updating instance properties, loading and defining spritesheets.
+
+### From json (people)
```

Comment:
For this an all other subsections that follow, remove the `(sprite)` from the title (people in this case). You already open each section with a clear codeblocked file reference, and the extra noise in the right navbar is unhelpful.